paper.md

```diff
@@ -300,7 +288,8 @@ This shows how Popper helps improve the performance of scientific workflows dras
 ### Workflow execution in Slurm clusters
 
 For this case study, we modified our training script to use the Horovod [@horovod] distributed deep learning framework to facilitate training with MPI [@gropp1999using] in a Slurm cluster.
-For running workflows in Slurm clusters, MPI supported container engines like Singularity, which is supported by popper need to be used.
+For running workflows in Slurm clusters, MPI supported container engines like Singularity, which is supported by Popper need to be used.
```

Comment:
needs a bit of fixing. These engines (Singularity, Charliecloud, Pyxis, etc.) work with Slurm, so they are both MPI- and Slurm-aware.